src/DataModel/IData.ts

```diff
@@ -0,0 +1,6 @@
+import {IMetadata} from "./IMetadata";
+
+export interface IData extends Object
```

Comment:
This library doesn't know what higher level libraries, apps, UIs want to do with the data so I wouldn't introduce such (leaky) separations/classifications here.